planning/development-strategy.md

```diff
@@ -20,6 +20,9 @@
 - [ ] *label* tag named add item
 - [ ] *input* tag type="text" for the user to enter an item
 - [ ] *add* button
+- [ ] *form* to have the ToDo list
+- [ ] *form* to have the Completed list
```

Comment:
We mentioned an input field in **_for: completed tasks_** (tag li with input type="checkbox", checked)

Reply:
My idea to have both forms existed at the interface of the program, whenever the user adds an item, the form of "toDo" will be increased @rayanejsilva 
![Screenshot from 2021-08-17 11-32-03](https://user-images.githubusercontent.com/28728040/129703764-81d4b92a-fb01-4487-a5f9-5621d033542f.png)

Reply:
I didn’t understand why we need a tag form for this. We can use a function to append the new items to our ul tag.